samples/core/GetStarted/Model.cs

```diff
@@ -8,10 +8,19 @@ public class BloggingContext : DbContext
         public DbSet<Blog> Blogs { get; set; }
         public DbSet<Post> Posts { get; set; }
 
+        public string DbPath { get; private set; }
+
+        public BloggingContext()
+        {
+            var folder = System.Environment.SpecialFolder.LocalApplicationData;
+            var path = System.Environment.GetFolderPath(folder);
+            DbPath = $"{path}\\blogging.db";
```

Comment:
Should it not be forward slash (or Env.Pathseparatir) ?

Reply:
Great catch! Submitted fix.